wiki/en/en-Setup.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: wiki
+title: "Setup"
+lang: "en"
+permalink: "/wiki/Setup"
+---
+
+# Setup
+
+To get the best from Jamulus, at a minimum you will need:
+
+1. **A wired internet connection** (not wi-fi - see the [FAQ](/wiki/FAQ))
+1. **Wired headphones** (not Bluetooth or speakers - see the [FAQ](/wiki/FAQ))
+1. **A reasonable audio device, sound card and/or mic** ([see this list](/kb/2021/01/05/Jamulus-Sound-Devices.html) for examples)
```

Comment:
... which is probably going to cause a lot of discussion:

Shouldn't we give a bit more information here or on the page we link. But this would then require it to be translated...

What I'd like to have somewhere:
1. Avoid internal sound cards (although you could start with it)
2. Only use devices which support 48kHz

Reply:
The primary mission of this page is to get the user connected and working. How **well** they are working is another matter and can be handled by troubleshooting, forums, software manual and all the rest of it. For a first time user, just to get the point of "MVP" is enough at this point. So - don't attempt to "optimise" them with advice on hardware purchases or will just seem like hard work. The link to the hardware list is enough, for example, if they are curious.

So what I was aiming for here was to just give instructions. Any _explanations_ for those instructions are elsewhere for those who want to read them if curious.

The 48kHz thing is arguable, but again - we're not expecting them to go dig out their manual on the off chance that it's not 48kHz. If it's not (and most are) then that's for later. Let's just make things easy.